plugins/user-settings/src/components/General/General.tsx

```diff
@@ -21,12 +21,12 @@ import { Profile } from './Profile';
 import { ThemeToggle } from './ThemeToggle';
 
 export const General = () => (
-  <Grid container spacing={3}>
-    <Grid item md={12}>
+  <Grid container direction="row" spacing={3}>
+    <Grid item sm={6}>
```

Comment:
In general when making grid items, do try them out in a smaller window size and add like `md={12}` or similar to make them stack vertically on constrained spaces